documentation.json

```diff
@@ -47,6 +47,11 @@
         "parameters": [],
         "description": "Returns a link to the KoalaBot Support Discord Server"
       }
+      {
+	"command": "setup",
+	"parameters": []
+	"description": "Allows access to configure the bot, once legal terms are agreed"
```

Comment:
please make sure the colon is kept in there, also need to add back in the comma on the line above (52)

Reply:
Done